e2e-jar-test/fips/src/test/java/net/snowflake/FipsIngestE2ETest.java

```diff
@@ -32,7 +31,6 @@ public void basicTest() throws InterruptedException {
   }
 
   @Test
-  @Ignore("Takes too long to run")
   public void longRunningTest() throws InterruptedException {
```

Comment:
:+1:

Reply:
Long running test should not be part of the merge gate, let's think about something else if we want the coverage for every PR

Reply:
Sounds good, I'm fine as long as it won't run hours as part of merge gate and block every PR